src/General.php

```diff
@@ -12,6 +12,57 @@ public function __construct()
         $this->files = $_FILES;

```

Comment:
wstrzyknij $_POST i $_FILES w konstruktorze ( ułatwi to ewentualne testowanie klasy ).